pkg/tests/api/plugins/api_plugins_test.go

```diff
@@ -113,6 +113,9 @@ func TestIntegrationPlugins(t *testing.T) {
 				require.Equal(t, tc.expStatus, resp.StatusCode)
 				b, err := io.ReadAll(resp.Body)
 				require.NoError(t, err)
+				// ignore info.version since it'll vary depending if plugins have been locally built
```

Comment:
I think we can make use of some of the `go-cmp` magic here instead. Something like:

```go
var result dtos.PluginList
err = json.Unmarshal(b, &result)
require.NoError(t, err)

expResp := expectedResp(t, tc.expRespPath) // this will also need to change to return dtos.PluginList

if diff := cmp.Diff(expResp, result, cmpopts.IgnoreFields(plugins.Info{}, "Version")); diff != "" {
	if updateSnapshotFlag {
		t.Log("updating snapshot results")
		var prettyJSON bytes.Buffer
		if err := json.Indent(&prettyJSON, b, "", "  "); err != nil {
			t.FailNow()
		}
		updateRespSnapshot(t, tc.expRespPath, prettyJSON.String())
	}
	t.Log("unexpected response", diff)
	t.FailNow()
}
```